### PR TITLE
Update CC & VA submission error alerts

### PR DESCRIPTION
**Pull request by LeKeve**
## Description
This PR updates copy in the appointment submission error alerts. It also updates the CC error alert to display parent site contact information.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#28256

## Testing done
- local, unit testing

### VA appointment error

Screen Shot 2021-08-09 at 8 43 15 AM

Screen Shot 2021-08-09 at 8 45 44 AM

### CC request error

Screen Shot 2021-08-09 at 8 42 27 AM

Screen Shot 2021-08-09 at 8 43 53 AM

## Acceptance criteria
- [x] CONTENT - If CC request submission technical error, Then alert content matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/review-details.md#errors) 
- [x] CONTENT - If CC request submission configuration error, Then alert content matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/review-details.md#errors) 
- [x] CONTENT - If VA request submission server error, Then alert content matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/review-details.md#errors) 
- [x] CONTENT - If VA request submission configuration error, Then alert content matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/review-details.md#errors) 
- [x] CONTENT - If VA direct submission technical error, Then alert content matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/review-details.md#errors) 
- [x] CONTENT - If VA direct submission configuration error, Then alert content matches the [copy doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/appointments/va-online-scheduling/content/copy-docs/review-details.md#errors) 
- [x] DESIGN - Designs match the specs (copy is FPO) 

### Design Assets
* [Sketch file](https://www.sketch.com/s/f97eb709-584d-4d37-a111-485828bcfafc/v/ER38wx/a/EL9OkmO/r/akoKpP#Inspector)

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/tests/new-appointment/components/ReviewPage/index.cc-request.unit.spec.jsx [hunk 1]
```diff
@@ -251,7 +251,7 @@ describe('VAOS <ReviewPage> CC request', () => {
     await screen.findByText('We couldn’t schedule this appointment');
 
     expect(screen.baseElement).contain.text(
-      'Something went wrong when we tried to submit your request and you’ll need to start over. We suggest you wait a day',
+      'Something went wrong when we tried to submit your request. You can try again later, or call your VA medical center to help with your request.',
```
Comment (jbalboni) [inline src/applications/vaos/tests/new-appointment/components/ReviewPage/index.cc-request.unit.spec.jsx:254]: Could we verify that the facility details are shown in these CC failure tests?

File: src/applications/vaos/new-appointment/components/ReviewPage/index.jsx [hunk 2]
```diff
@@ -95,33 +94,21 @@ export default function ReviewPage() {
...
-                      data.vaFacility || data.communityCareSystemId,
-                    )}`}
-                  >
-                    {submitStatusVaos400
-                      ? 'Find facility contact information'
-                      : 'Contact your local VA medical center'}
-                  </NewTabAnchor>
-                )}
-                {!!facilityDetails && (
+                {(!!facilityDetails || parentFacility) && (
```
Comment (jbalboni) [inline src/applications/vaos/new-appointment/components/ReviewPage/index.jsx:108]: Given that we're repeating the `|| ` failover a few times in this file, could be worth creating a variable up at the start of the function that does the fallback once?

**Review by jbalboni — COMMENTED** [review]
Couple minor things, looks good overall

File: src/applications/vaos/new-appointment/redux/selectors.js [hunk 3]
```diff
@@ -121,10 +121,6 @@ export function getSiteIdForChosenFacility(state) {
   return getSiteIdFromFacilityId(getFormData(state).vaFacility);
 }
 
-export function getChosenFacilityDetails(state) {
```
Comment (LeKeve) [inline src/applications/vaos/new-appointment/redux/selectors.js:124]: Removed this method since it just calls another selector method.

**Review by jbalboni — APPROVED** [review]

**Pull request merged**